notebooks/VAD

```diff
@@ -0,0 +1,37 @@
+from scipy import signal
+# input : y - the signal, sr - sample rate
+# output : start_times - vector of start times of the sound events
+#          end_time - vector of end times of the sound events            
+def detect_activity(y, sr,
+        n_mels=128, fmin=1000, fmax=11025, hop_length=512,
+        gain=0.8, bias=10, power=0.25, pcen_time_constant=0.06, eps=1e-06,
+        medfilt_time_constant=1, normalized =true,
+        peak_threshold=0.45, activity_threshold=0.2):
+    # 1. compute mel-frequency spectrogram
+    # (call librosa melspec)
+    melspec = librosa.feature.melspectrogram(y, sr=sr, fmin=fmin, fmax=fmax, n_mels=n_mels)
+    logmelspec = librosa.power_to_db(melspec)
+    
+    # 2. compute PCEN
+    # (call librosa PCEN)
+    pcen = librosa.core.pcen(melspec, sr=sr,
+    gain=gain, bias=bias, power=power, time_constant=pcen_time_constant, eps=eps)
```

Comment:
make sure to standardize your usage of whitespace around the equal sign
see PEP8